### PR TITLE
Liket (ልኬት)

### PR DESCRIPTION
**Pull request by IamYunusAli**
In this pull request, I have implemented the following things.

- Follow SPA best practices to implement the project.
- Used React and Redux efficiently.
- Retrieved from the API should be stored in the Redux store.
- Filter the data you retrieve from the API using a Filter stateless component.
- Implement ID-based routing for each page item.
- Test the project using React testing Library.
-  Full Description of the project is found here [Loom video](https://www.loom.com/share/d965563ae4bf4aaf8946dc06764aa867)
- Deployed link can be found [here](https://liket.netlify.app/)

### CODE REVIEW

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @IamYunusAli , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪

- project works fine
- all tests passes
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/style/coin.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.list-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  list-style-type: none;
+  background-color: #4369b2;
+  color: #fff;
+  padding-left: 5px;
+  flex-wrap: wrap;
+}
```
Comment (mukhammadrizooff) [inline src/components/style/coin.css:39]: - [ ] [REQUIRED] Please fix your style issues!
- You have made a mistake in setting the background color of the cells, please recheck the given Behance design and solve them!  

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @IamYunusAli, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/style/coin.css (hunk 1 above)
Comment (IamYunusAli) [reply src/components/style/coin.css]: @mukhammadrizooff thank you for the feedback even though I haven't seen any requirement on the project description I have implemented it
new one

**Review by zhadier — APPROVED** [review]
## Hi @IamYunusAli  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**